### PR TITLE
PsiTransfer

### PR DESCRIPTION
**Pull request by teulaert**
Add PsiTransfer to the Quickstarters. A simple open source self-hosted file sharing solution. It's an alternative to paid services like Dropbox, WeTransfer.

### CODE REVIEW

File: rocketchat/README.md [hunk 1]
```diff
@@ -5,9 +5,21 @@ Open Source Slack Clone based on Meteor and MongoDB.
...
 
 Example:
-   
-sloppy start -var=URI:coolchat.sloppy.zone rocketchat.json
-```
\ No newline at end of file
+```   
+$ sloppy start -var=URI:coolchat.sloppy.zone rocketchat.json
+```
+
```
Comment (FrontEndCoffee) [inline rocketchat/README.md:15]: These Rocketchat related changes should not be in this PsiTransfer PR

File: rocketchat/rocketchat.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
           "domain": {
             "uri": "$URI"
           },
-          "mem": 512,
+          "mem": 768,
           "image": "rocketchat/rocket.chat:latest",
           "instances": 1,
           "volumes": [
```
Comment (FrontEndCoffee) [inline rocketchat/rocketchat.json:15]: These Rocketchat related changes should not be in this PsiTransfer PR

File: rocketchat/rocketchat.yml [hunk 3]
```diff
@@ -1,26 +0,0 @@
-version: "v1"
-project: "rocketchat"
-services:
-  frontend:
```
Comment (FrontEndCoffee) [inline rocketchat/rocketchat.yml:4]: These Rocketchat related changes should not be in this PsiTransfer PR

**Review by FrontEndCoffee — CHANGES_REQUESTED** [review]
It seems that something has gone wrong with your branching, the `psitransfer` branch also contains the changes for you earlier `rocketchat` branch. Please remove these changes from this branch.

I have added a comment on all three of the files for your convenience

**Review by timmakken — APPROVED** [review]

**Pull request merged**